packages/netlify-cms-backend-bitbucket/src/AuthenticationPage.js

```diff
@@ -80,6 +80,9 @@ export default class BitbucketAuthenticationPage extends React.Component {
           <React.Fragment>
             <LoginButtonIcon type="bitbucket" />
             {inProgress ? 'Logging in...' : 'Login with Bitbucket'}
+            {config.get('site_url') ? (
```

Comment:
These can be shrunk down: `{config.get('site_url') && <GoBackButton href={config.get('site_url')}/>}`